package.json

```diff
@@ -130,8 +130,8 @@
     "webpack-dev-server": "2.11.2"
   },
   "dependencies": {
-    "algoliasearch": "3.25.1",
-    "algoliasearch-helper": "2.24.0",
+    "algoliasearch": "3.27.0",
+    "algoliasearch-helper": "2.25.1",
```

Comment:
I wonder why we pinned dependencies to patch versions here 🤔 I know we have renovate, but it can cause duplication if an enduser has those in their dependencies too with another version specified

Reply:
We did this change before we all agree that we should go for pinned dependencies only for the dev one. We should revert the production dependencies to caret range. We can do it in an another PR.